### PR TITLE
Update Basic metadata profile for models for functionality 

### PR DESCRIPTION
**Pull request by shahchiragh**
Changes include issues mentioned at https://github.com/usgs/modelcatalog-tam/issues

### CODE REVIEW

**Comment by brandonserna** [comment]
@shahchiragh thanks for the PR! 👏 

Looking pretty good! A couple requests:

Continue naming conventions for the attributes in snake case (example: process_based = "Process-based"). I believe this is not valid python for the key as it might evaluate as `process(var1)`-(minus)`based(var2) = "Process-based"` 
see comment on l:19

Remove `.DS_Store ` from commit - you can add these to your `.gitignore` to make it easier

Are you able to modify the [tests](https://github.com/usgs/modelcatalog-tam/blob/main/tests/test_basicmetadata.py#L70) to reflect this new enumeration? If not lmk and I can after

File: metadata_schema/SoftwareSchema.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+from pydantic import BaseModel, HttpUrl
+from typing import List, Optional
+
+
+class Softwares(BaseModel):
```
Comment (brandonserna) [inline metadata_schema/SoftwareSchema.py]: Can we name this class "Software"?

Comment (shahchiragh) [reply metadata_schema/SoftwareSchema.py]: Sure, I can update it to only Software.

**Comment by hsu000001** [comment]
Are we able to make missionarea pull from a new enumerated list?
usgs_missionarea: Optional[List[str]]
usgs_missionarea: Optional[List[MissionAreaKeywordEnum]]

Desire the following options:
Core Science Systems
Ecosystems
Energy and Minerals
Natural Hazards
Water Resources

class MissionAreaKeywordEnum(str, Enum):
    core_science_systems = "Core Science Systems"
    ecosystems = "Ecosystems"
    energy_minerals = "Energy and Minerals"
    natural_hazards = "Natural Hazards"
    water = "Water Resources"

**Comment by brandonserna** [comment]
LGTM! ✅

**Pull request merged**